app/components/GitHub.tsx

```diff
@@ -0,0 +1,12 @@
+import Image from 'next/image'
+
+export default function GitHubButton() {
+    return (
+        <button
+            onClick={() => window.open('https://github.com/felixhir/ticket-generator', '_blank')}
+            className='absolute left-4 bottom-4 transition-transform transform hover:scale-110 cursor-pointer dark:bg-white rounded-full'
```

Comment:
While I like having an additional component, I tend to advocate having the positioning done by the component utilizing the button. This way, it becomes more reusable. Although I doubt we want to use this more than once, so I would be fine making an exception here :)

Reply:
Yep that's totally correct. I intent to make the whole page responsive in another PR, so this will probably change anyways then